### ISSUE TITLE
tsconfig strictNullChecks

### ISSUE THREAD

**Issue by ryanhyslop** [MrFrankel/ngx-popper#5]
Including in a project with `strictNullChecks:true` in the `tsconfig.json` the following error is returned on compile:

```
ERROR in node_modules/ngx-popper/src/popper-content.ts (210,7): Object is possibly 'undefined'.

ERROR in node_modules/ngx-popper/src/popper-content.ts (210,7): Type '{ boundariesElement: Element | null; }' is not assignable to type '(BaseModifier & { priority?: Position[] | undefined; padding?: number | undefined; boundariesElem...'.
  Type '{ boundariesElement: Element | null; }' is not assignable to type 'BaseModifier & { priority?: Position[] | undefined; padding?: number | undefined; boundariesEleme...'.
    Type '{ boundariesElement: Element | null; }' is not assignable to type '{ priority?: Position[] | undefined; padding?: number | undefined; boundariesElement?: "window" |...'.
      Types of property 'boundariesElement' are incompatible.
        Type 'Element | null' is not assignable to type '"window" | Element | "scrollParent" | "viewport" | undefined'.
          Type 'null' is not assignable to type '"window" | Element | "scrollParent" | "viewport" | undefined'.
```

Line 210 can be refactored to support strictNullChecks, or perhaps typings could be defined separately rather than from the source code? But just wondering if there was any thoughts before I potentially submitted a PR?

**Comment by MrFrankel** [comment]
@ryanhyslop you can PR away. if we can supports strictNullChecks easily then why not?
Thanks for bringing it up :)

**Comment by MrFrankel** [comment]
Fixed by @ryanhyslop at #6
